### DISCUSSION TITLE
Force Update Feature Request

### DISCUSSION THREAD

**Discussion by tboy1337** [masterking32/MasterCryptoFarmBot#60] [Ideas]
**Is your feature request related to a problem? Please describe.**

```
2024-10-13 11:56:48 [MasterCryptoFarmBot] [11:56:48] [ERROR] 🔄 Main Project is not updated ...
2024-10-13 11:56:50 [MasterCryptoFarmBot] [11:56:50] [ERROR] 🔄 Please restart the bot ...
```

![error](https://github.com/user-attachments/assets/47205621-a55d-4955-9c60-bb99da6e863d)

**Describe the solution you'd like**
Please add the ability to force check for an update for the framework.

**Describe alternatives you've considered**
n/a

**Additional context**
n/a

**Comment by masterking32** [comment]
There is another updater in the starter files, so if one fails, the second one will work as well, Just need to restart the project from the control panel.

**Reply by tboy1337** [reply]
I just mean a button the user can click to force it to check for an update.
